diff --git a/node_modules/react-native-pitch-detector/lib/commonjs/internal/pitch-detector/index.js b/node_modules/react-native-pitch-detector/lib/commonjs/internal/pitch-detector/index.js
index 4287998..1383d1a 100644
--- a/node_modules/react-native-pitch-detector/lib/commonjs/internal/pitch-detector/index.js
+++ b/node_modules/react-native-pitch-detector/lib/commonjs/internal/pitch-detector/index.js
@@ -84,10 +84,6 @@ class InternalPitchDetector {
   async start(config) {
     try {
       var _this$module2;
-      const permission = await this.hasPermissions();
-      if (!permission) {
-        throw new _erros.PitchDetectorError(_types.PitchDetectorErrors.PERMISSIONS_ERROR);
-      }
       const configuration = (0, _utils.merge)(this.getDefaultConfig(), config ?? {});
       const params = _reactNative.Platform.select({
         android: configuration.android,
